keras_hub/src/models/yolo_v8/yolo_v8_backbone_presets.py

```diff
@@ -0,0 +1,104 @@
+backbone_presets_no_weights = {
+    "yolo_v8_xs_backbone": {
```

Comment:
Just FYI: they presets will need to be generated again

Reply:
Yes exactly! I am going at the moment through your comments and I will upload the new presets and update the old ones. I might have some questions about that but I will let you know here.